packages/contracts-bedrock/README.md

```diff
@@ -68,7 +68,7 @@ The spec for the deploy config is defined by the `deployConfigSpec` located insi
 ### Execution
 
 1. Set the env vars `ETH_RPC_URL`, `PRIVATE_KEY` and `ETHERSCAN_API_KEY` if contract verification is desired
-1. Deploy the contracts with `forge script -vvv scripts/Deploy.s.sol:Deploy --rpc-url $ETH_RPC_URL --broadcast --private-key $PRIVATE_KEY`
+1. Deploy the contracts with `forge script -vvv scripts/Deploy.s.sol:Deploy --rpc-url $ETH_RPC_URL --sig 'runWithStateDiff()' --broadcast --private-key $PRIVATE_KEY`
```

Comment:
Does deployment actually need to use `runWithStateDiff` instead of `run`?

Reply:
Removed and cleared up the readme.